setup.py

```diff
@@ -14,7 +14,7 @@
     py_modules=['pytest_allure_dsl'],
     install_requires=[
         'pytest',
-        'pytest-allure-adaptor==1.7.10',
+        'git+https://github.com/oztqa/allure-pytest.git@1.7.11',
```

Comment:
error in pytest-allure-dsl setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Invalid requirement, parse error at "'+https:/'"